Code/Framework/AzToolsFramework/Platform/Linux/AzToolsFramework/API/PythonLoader_Linux.cpp

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzToolsFramework/API/EditorPythonConsoleBus.h>
+#include <dlfcn.h>
+
+namespace AzToolsFramework::EmbeddedPython
+{
+    class PythonLoader
+    {
+    public:
+        PythonLoader()
+        {
+            m_embeddedLibPythonHandle = dlopen("libpython3.7m.so.1.0", RTLD_NOW | RTLD_GLOBAL);
```

Comment:
Is there anyway we grab this from a` #define`

Reply:
It would be nice to couple it with the 3rdParty::Python module somehow, but i'm not sure if we want to bring that in as a dependency for AzToolsFramework.

Reply:
Actually, I will run into issues that requires loading more than just libpython3.7m.so.1.0. For pyside, because the shared libraries are distributed in different locations, it will need to dlopen trick for them as well. So i'm not sure if we can generalize it this way (or if its worth it)